README.md

```diff
@@ -76,12 +83,21 @@ async function connectToWhatsApp () {
             console.log('opened connection')
         }
     })
+
+    // you need to save your credentials for when it restart the connection.
+    sock.ev.on('creds.update', async () => {
+        await saveCreds()
+    });
+
+    // this will send a message to yourself 5 seconds after you log in.
     sock.ev.on('messages.upsert', m => {
         console.log(JSON.stringify(m, undefined, 2))
 
         console.log('replying to', m.messages[0].key.remoteJid)
-        await sock.sendMessage(m.messages[0].key.remoteJid!, { text: 'Hello there!' })
+        setTimeout(() => sock.sendMessage(m.messages[0].key.remoteJid!, { text: 'Hello there!' }), 5000);
     })
+
+    // setTimeout(() => sock.sendMessage('556185560820@s.whatsapp.net', { text: 'mensagem automatizada, iphone é coisa de chupa rola' }), 5000)
```

Comment:
~brazil mentioned :brazil:~
remove this line, please and also make indentation standardized using only 4 spaces (not tabs).

Reply:
funny thing you are a brazillian and understood my comment =]

about the indentation

1. I see a prettierignore but not a prettierrc, maybe that would prevent the problem
2. fixed.

Reply:
I do use eslint for code formatting as well, they say it is faster with prettier but configuring only one is just more comfortable.